examples/layers/widgets/spinning_square.dart

```diff
@@ -24,6 +24,12 @@ class _SpinningSquareState extends State<SpinningSquare> {
   }
 
   @override
+  void dispose() {
+    _animation.stop();
```

Comment:
I thought we agreed (as a convention) to use _animation.dispose() in this situation?